### PR TITLE
Don't show tabwidget as tab

### PR DESCRIPTION
**Pull request by Montel**

### CODE REVIEW

File: internal/common/enums.rs [hunk 1]
```diff
@@ -341,6 +341,8 @@ macro_rules! for_each_enums {
                 Spinbox,
                 /// The element is a [`Tab`](../widgets/tabwidget.md) or behaves like one.
                 Tab,
+                /// The element is a [`TabList`](../widgets/tabwidget.md) or behaves like one.
+                TabList,
```
Comment (ogoffart) [inline internal/common/enums.rs:345]:
```suggestion
                /// The element is a [`TabWidget`](../widgets/tabwidget.md) or behaves like one.
                TabList,
```

But even then , this is more like the tabbar.

```suggestion
                /// The element is similar to the tab bar in a [`TabWidget`](../widgets/tabwidget.md).
                TabList,
```

**Review by ogoffart — APPROVED** [review]
I feel the `tab-list` name is a bit wierd, i'd have named it `tab-bar`.   But since that's the name used by all the accessibility interfaces (including web) i guess it's fine.  @tronical : agree?

**Comment by tronical** [comment]
Agreed :)

**Comment by tronical** [comment]
Could it be that your patch its missing the Cupertino style?

**Comment by Montel** [comment]
indeed missing. I will provide a patch soon

**Pull request merged**